target/linux/mvebu/files-4.19/arch/arm64/boot/dts/marvell/armada-3720-espressobin-ultra.dts

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
```

Comment:
The file in files-4.19 can be removed, since the one in "files" will work for both kernels. (unless you have specific differences, then you would need files-4.19 and files-5.4 ...)

Reply:
Removed dts from files-4.19.

Reply:
File is still there. Did you forget to push?